README.md

```diff
@@ -15,8 +15,12 @@ Hazzat backend REST API writen in vanilla Node.js with a React frontend. [www.ha
 
```

Comment:
I know this is unrelated to this change, but could you please also add a section for development and CI/CD?  pull requests should be into develop.  Commits to Develop will deploy to https://hazzat-api-ppe.azurewebsites.net and master will deploy to https://api.hazzat.com

Reply:
I think we’ll need a new CONTRIBUTING.md at this point.

Reply:
ok

Reply:
done